azuredeploy.parameters.json

```diff
@@ -7,6 +7,7 @@
                 "webServerType":               { "value": "apache" },
                 "dbServerType":                { "value": "mssql" },
                 "fileServerDiskCount":         { "value": 2 },
-                "fileServerDiskSize":          { "value": 32 }
+                "fileServerDiskSize":          { "value": 32 },
+		"azureSearchSwitch":           { "value": true }
```

Comment:
Again, the param values in this file will dictate how the Azure/azure-quickstarts-templates repo will perform its test deployment. Yes, it'd be good to test all the available features in the CI, but then it'll easily exceed the 2-hour time limit, and I think we should minimize the enabled options there. So please turn this off to false.

And also the indentation. Not sure if there are tabs being involved. We should use just spaces only. Please let us know if the original indentations already included tabs. It'd be great if you fix all of them in that case.